TODO

```diff
@@ -0,0 +1 @@
+* look into profile updates to remove blank values
```

Comment:
:eyes: did you accidentally commit this @citizen428 ?

Reply:
Yes, I did 🤦 It's my private TODO list...